samples/Mvc.Server/Startup.cs

```diff
@@ -34,7 +34,10 @@ public class Startup {
 
             services.AddIdentity<ApplicationUser, IdentityRole>()
                 .AddEntityFrameworkStores<ApplicationDbContext>()
-                .AddDefaultTokenProviders();
+                .AddDefaultTokenProviders()
+
+                .AddOpenIddict<ApplicationUser, Application>()
```

Comment:
why not `AddOpenIddict<Application>` ApplicationUser is already part of the AddIdentity part, can we use that ?